src/components/PhoneNumberInput/style.css

```diff
@@ -11,9 +13,10 @@
 }
 
 .phoneNumberContainer {
+  @base-width: 294;
   float: left;
   margin: 8px 12px;
-  width: 294px;
+  width: 100%*(@base-width/@parent-width);
```

Comment:
This percentage applies to the parent so you end up having X% of X% rather than X%.
<img width="1440" alt="screen shot 2017-12-01 at 12 12 21" src="https://user-images.githubusercontent.com/6737066/33482528-6166573a-d691-11e7-91bd-262c28075879.png">

Reply:
True, I've fixed it.